### PR TITLE
Update packages to latest versions

### LINKED ISSUE #28: Update node-stubs to latest packages?
Hi, I have noticed that many of the node-stubs packages have updated versions.  How often does this package synchronize with the latest versions of the browserified node packages?

If you want an update on these, let me know and I can pull together a PR.

### PR DESCRIPTION
**Pull request by brucejo75**
- [x] Update all packages to their very latest versions.
- [x] Bump package version

**Note:** my only testing was to run up my project and make sure it works...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,37 +2,37 @@
   "name": "meteor-node-stubs",
   "author": "Ben Newman <ben@meteor.com>",
   "description": "Stub implementations of Node built-in modules, a la Browserify",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (StorytellerCZ) [inline package.json]: Since we have major version updates here it might be a good idea to have at least minor version bump.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

**Comment by StorytellerCZ** [comment]
As for testing once we publish I'll update it let's make an announcement on the forums for people to test it before pushing to production.

File: package.json (hunk 1 above)
Comment (brucejo75) [reply package.json]: @StorytellerCZ, OK, will move to `1.1.0`

**Comment by StorytellerCZ** [comment]
I'm going to hold on for this after Meteor 2.3 release so that people have enough time to handle changes in there before we jump on them with this. Probably won't change much, but better take it slowly.

**Comment by brucejo75** [comment]
Sounds right.

**Comment by brucejo75** [comment]
@StorytellerCZ, is this ready to merge?

**Comment by StorytellerCZ** [comment]
Yes! Time to do this!

**Review by StorytellerCZ — APPROVED** [review]

**Pull request merged**